web/src/features/charts/graphUtils.ts

```diff
@@ -211,30 +211,41 @@ export function getElectricityProductionValue({
   return generationTypeStorage === 0 ? 0 : -generationTypeStorage;
 }
 
-function analyzeChartData(chartData: AreaGraphElement[]) {
+function analyzeChartData(chartData: AreaGraphElement[], timeAverage: TimeAverages) {
   let estimatedCount = 0;
   let tsaCount = 0;
+  let allEstimated = true;
+  const isHourly = timeAverage === TimeAverages.HOURLY;
   for (const chartElement of chartData) {
     if (chartElement.meta.estimationMethod === EstimationMethods.TSA) {
       tsaCount++;
     }
     if (chartElement.meta.estimatedPercentage || chartElement.meta.estimationMethod) {
       estimatedCount++;
     }
+    if (
+      (!isHourly && chartElement.meta.estimatedPercentage !== 100) ||
+      (isHourly && !chartElement.meta.estimationMethod)
+    ) {
+      allEstimated = false;
+    }
   }
   return {
     allTimeSlicerAverageMethod: tsaCount === chartData.length,
-    allEstimated: estimatedCount === chartData.length,
-    hasEstimation: estimatedCount > 0,
+    allEstimated,
+    hasEstimation: isHourly ? allEstimated : estimatedCount > 0,
```

Comment:
I would prefer if this was renamed, to represent the new state.

Now it rather means all estimated or has estimations.

Not sure what a good name is tho.